units/tmp.mount.m4

```diff
@@ -6,7 +6,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Temporary Directory
+Description=/tmp directory
```

Comment:
Hmm, I'd prefer if we would stick to uppercasing "Directory", we do that for all other dirs. Also, /tmp is pretty low-levelish, and I think we should stick to slightly higher-level explanations for this. my suggesting would hence be:

```Temporary Directory (/tmp)```

or so. That explains what is meant in a high-level way, but then becomes more explanatory and precise with the suffix...